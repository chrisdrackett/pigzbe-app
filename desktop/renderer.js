import {AppRegistry} from 'react-native';
import App from './bundle.web.js';

AppRegistry.registerComponent('App', () => App);
AppRegistry.runApplication('App', {rootTag: document.getElementById('react-root')});
