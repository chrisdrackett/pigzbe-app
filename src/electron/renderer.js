import {AppRegistry} from 'react-native';
// import App from '../../App';
import App from '../../dist/bundle.web.js';

AppRegistry.registerComponent('App', () => App);
AppRegistry.runApplication('App', {rootTag: document.getElementById('react-root')});
