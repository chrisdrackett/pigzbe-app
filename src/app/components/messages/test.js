import React from 'react';
import renderer from 'react-test-renderer';
import {MessagesComponent} from './';

const props = {
    dispatch: () => {},
    messages: [{
        key: '1',
        date: '2018-04-20T12:00+01:00',
        text: 'Morbi in sem quis dui placerat ornare. Pellentesque odio nisi, euismod in, pharetra a, ultricies in, diam. Sed arcu. Cras consequat.',
        link: 'https://pigzbe.com'
    }, {
        key: '2',
        date: '2018-04-20T09:00+01:00',
        text: 'Phasellus ultrices nulla quis nibh. Quisque a lectus. Donec consectetuer ligula vulputate sem tristique cursus. Nam nulla quam, gravida non, commodo a, sodales sit amet, nisi.'
    }],
    loading: false,
    error: null
};

describe('Messages', () => {
    // test('renders correctly with messages', () => {
    //     const tree = renderer.create(<MessagesComponent {...props}/>).toJSON();
    //     expect(tree).toMatchSnapshot();
    // });

    test('renders correctly without messages', () => {
        const tree = renderer.create(
            <MessagesComponent {...Object.assign({}, props, {
                messages: []
            })}/>
        ).toJSON();
        expect(tree).toMatchSnapshot();
    });
});
