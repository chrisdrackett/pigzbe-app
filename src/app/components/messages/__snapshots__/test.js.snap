// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Messages renders correctly without messages 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "alignSelf": "stretch",
      "backgroundColor": "rgb(0, 50, 120)",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "flex": 1,
        "flexBasis": 162,
        "flexGrow": 0,
        "height": 162,
        "justifyContent": "flex-start",
        "width": "100%",
      }
    }
  >
    <Image
      source={
        Object {
          "testUri": "../../../src/app/components/logo/images/pigzbe_logo_inapp.png",
        }
      }
      style={
        Object {
          "height": 25,
          "marginBottom": 10,
          "marginTop": 34,
          "width": 65,
        }
      }
    />
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "rgb(255, 255, 255)",
          "fontFamily": "Poppins",
          "fontSize": 16,
          "fontWeight": "bold",
          "marginLeft": 20,
          "marginRight": 20,
          "marginTop": 10,
          "textAlign": "center",
        }
      }
    >
      Your Updates
    </Text>
    <View
      style={
        Object {
          "alignItems": "center",
          "bottom": 0,
          "flex": 1,
          "justifyContent": "flex-end",
          "position": "absolute",
        }
      }
    >
      <Image
        source={
          Object {
            "testUri": "../../../src/app/components/pig/pig.png",
          }
        }
        style={
          Object {
            "height": 35,
            "width": 81,
          }
        }
      />
    </View>
  </View>
  <View
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "rgb(247, 247, 250)",
        "flex": 1,
        "justifyContent": "center",
        "width": "100%",
      }
    }
  >
    <View
      style={
        Object {
          "backgroundColor": "rgba(0, 0, 0, 0.05)",
          "height": 5,
          "left": 0,
          "position": "absolute",
          "right": 0,
          "top": 0,
          "zIndex": 1,
        }
      }
    />
    <RCTScrollView
      data={Array []}
      disableVirtualization={false}
      getItem={[Function]}
      getItemCount={[Function]}
      horizontal={false}
      initialNumToRender={10}
      invertStickyHeaders={undefined}
      keyExtractor={[Function]}
      maxToRenderPerBatch={10}
      numColumns={1}
      onContentSizeChange={[Function]}
      onEndReachedThreshold={2}
      onLayout={[Function]}
      onMomentumScrollEnd={[Function]}
      onScroll={[Function]}
      onScrollBeginDrag={[Function]}
      onScrollEndDrag={[Function]}
      renderItem={[Function]}
      scrollEventThrottle={50}
      stickyHeaderIndices={Array []}
      updateCellsBatchingPeriod={50}
      viewabilityConfigCallbackPairs={Array []}
      windowSize={21}
    >
      <View />
    </RCTScrollView>
  </View>
</View>
`;
